spec/features/users/user_can_invite_caretaker_spec.rb

```diff
@@ -4,30 +4,39 @@
   it 'shows a button to add a caretaker to a garden on the gardens index', :vcr do
     user = create(:user)
     allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
-    garden_1 = create(:garden, user: user, name: "Frontyard")
-    garden_2 = create(:garden, user: user, name: "Backyard")
+    garden_1 = create(:garden, owners: [user], name: "Frontyard")
+    garden_2 = create(:garden, owners: [user], name: "Backyard")
 
     visit gardens_path
 
     within "#garden-#{garden_1.id}" do
       expect(page).to have_link("Add Caretaker")
       click_link "Add Caretaker"
-      expect(current_path).to eq(invite_path)
+      expect(current_path).to eq(invite_path(garden_1))
     end
   end
 
   it 'can send an invitation to become a caretaker', :vcr do
-    user = create(:user)
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
-    garden_1 = create(:garden, user: user, name: "Frontyard")
+    user_1 = create(:user)
+    user_2 = create(:user)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user_1)
+    garden_1 = create(:garden, owners: [user_1], name: "Frontyard")
+    clear_emails
 
     visit gardens_path
 
     click_link "Add Caretaker"
-    fill_in :email, with: "ali.benetka@gmail.com"
+    email = "ali.benetka@gmail.com"
```

Comment:
@abenetka Do you want your personal email in here? This could just be generic e.g. user@example.com